grails-app/views/tag/create.gsp

```diff
@@ -19,66 +19,72 @@
+            <g:render template="summary"/>
 
-                            <tr class="prop">
-                                <td valign="top" class="name">
-                                    <label for="createdBy"><warehouse:message code="tag.createdBy.label" default="Created By" /></label>
-                                </td>
-                                <td valign="top" class="value ${hasErrors(bean: tagInstance, field: 'createdBy', 'errors')}">
-                                    <g:select name="createdBy.id" from="${org.pih.warehouse.core.User.list()}" optionKey="id" value="${tagInstance?.createdBy?.id}" noSelection="['null': '']" />
-                                </td>
-                            </tr>
+            <g:form action="save" method="post" >
+                <div class="box">
+                    <h2><g:message code="default.create.label" args="[g.message(code:'tag.label')]" /></h2>
 
-                            <tr class="prop">
-                                <td valign="top" class="name">
-                                    <label for="dateCreated"><warehouse:message code="tag.dateCreated.label" default="Date Created" /></label>
-                                </td>
-                                <td valign="top" class="value ${hasErrors(bean: tagInstance, field: 'dateCreated', 'errors')}">
-                                    <g:datePicker name="dateCreated" precision="minute" value="${tagInstance?.dateCreated}"  />
-                                </td>
-                            </tr>
+                    <div class="dialog">
+                        <table>
+                            <tbody>
 
-                            <tr class="prop">
-                                <td valign="top" class="name">
-                                    <label for="lastUpdated"><warehouse:message code="tag.lastUpdated.label" default="Last Updated" /></label>
-                                </td>
-                                <td valign="top" class="value ${hasErrors(bean: tagInstance, field: 'lastUpdated', 'errors')}">
-                                    <g:datePicker name="lastUpdated" precision="minute" value="${tagInstance?.lastUpdated}"  />
-                                </td>
-                            </tr>
+                                <tr class="prop">
+                                    <td valign="top" class="name">
+                                        <label for="tag"><warehouse:message code="tag.tag.label" default="Tag" /></label>
+                                    </td>
+                                    <td valign="top" class="value ${hasErrors(bean: tagInstance, field: 'tag', 'errors')}">
+                                        <g:textField name="tag" size="80" class="text medium" value="${tagInstance?.tag}" />
+                                    </td>
+                                </tr>
 
+%{--                                <tr class="prop">--}%
```

Comment:
Is the commented-out part still required?

Reply:
Probably not. I just wanted to be able to quickly revert if someone complains about it.